pkg/coreapi/graph/resolvers/function.go

```diff
@@ -12,3 +12,13 @@ func (r *functionResolver) App(ctx context.Context, obj *models.Function) (*cqrs
 	appID := uuid.MustParse(obj.AppID)
 	return r.Data.GetAppByID(ctx, appID)
 }
+
+func (qr *queryResolver) FunctionBySlug(ctx context.Context, query models.FunctionQuery) (*models.Function, error) {
+	dummyLocalWorkspaceID := uuid.MustParse("6c6f6361-6c00-0000-0000-000000000000")
```

Comment:
Might be good to use this:
https://github.com/inngest/inngest/blob/main/pkg/consts/devserver.go#L33

Sometimes things get weird when you don't use it. Also, it'll help discovery when we eventually add multi-environment support to the Dev Server

Reply:
cool didn't know about that, my joke of encoding "local" in hex and having it at the front of the UUID is toast 😆

Reply:
Ha, I didn't notice that!